playwright/tests/script-runner/debug.spec.ts

```diff
@@ -138,20 +138,25 @@ test('displays the call stack', async ({ page }) => {
   await expect(page.locator('text=Show Call Stack')).toHaveAttribute('disabled', 'disabled')
 })
 
-test('displays disconnect icon', async ({ page }) => {
+test.only('displays disconnect icon', async ({ page }) => {
```

Comment:
Not sure you want to include this.

Reply:
Agree, remove the only and I think looks good.